### PR TITLE
adding nl dir for nlc

### PR DESCRIPTION
**Pull request by jdicarl**
Added nl files for NLC. Renamed all `index.md` files to `getting-started.md`

### CODE REVIEW

File: services/natural-language-classifier/nl/de/toc [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{:navgroup: .navgroup}
+{:topicgroup: .topicgroup}
+
+{: .toc subcollection="natural-language-classifier" audience="service" href="/docs/services/natural-language-classifier/index.html"}
```
Comment (SirSpidey) [inline services/natural-language-classifier/nl/de/toc]: each of the toc files needs this line changed to `href="/docs/services/natural-language-classifier/getting-started.html"` to match the file name.

File: services/natural-language-classifier/nl/de/toc [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Natural Language Classifier
+
+    {: .navgroup id="learn"}
+    index.md
```
Comment (SirSpidey) [inline services/natural-language-classifier/nl/de/toc]: also, `index.md` needs to change in each toc to `getting-started.md`

**Comment by jdicarl** [comment]
@SirSpidey I made another commit to fix the `toc` files. Can you review again? I wasn't sure how to do a re-review, so I just added you again as the review. Let me know if I should've done it differently.

**Comment by SirSpidey** [comment]
@jdicarl I requested an NL build at https://releaseblueprints.ibm.com/display/CLOUDOE/Doc+build+requests+queue#Docbuildrequestsqueue-Promotingstagingcontenttoproductiontable. I hope to be notified when the build is complete

**Pull request merged**